dev/TabView/TabView.idl

```diff
@@ -178,7 +178,10 @@ unsealed runtimeclass TabViewItemTemplateSettings : Windows.UI.Xaml.DependencyOb
 
     Windows.UI.Xaml.Controls.IconElement IconElement;
 
+    Windows.UI.Xaml.Media.Geometry TabGeometry;
```

Comment:
Please work with Gabby for API review/doc update for this new property.

Reply:
Will do.